### DISCUSSION TITLE
The resource <URL> was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.

### DISCUSSION THREAD

**Discussion by 1alexvash** [vercel/next.js#41447] [Help]
Is it even possible to fix this warning using Next image?
![image](https://user-images.githubusercontent.com/46556479/196004112-66b91ee5-87e0-45b6-bb8b-149f0ffcbddc.png)

It is annoying to have something distracting in the console

My code | https://github.com/1alexvash/AlexCodeBlog

```js
<Link href={`/post/${post.slug}`} rel="prefetch" as="image">
  <a className="post-img">
    <Image
      src={post.featuredImage ?? "/post-images/placeholder.png"}
      alt="blog post image"
      layout="fill"
      objectFit="cover"
      priority={index === 0}
    />
  </a>
</Link>
```

**Comment by icyJoseph** [comment]
Isn't this a browser warning? https://github.com/vercel/next.js/issues/6517#issuecomment-469063892

Pretty much Chrome is complaining that it invested time prefetching that resource but it was not used within a certain heuristic time, or never at all. 

The root issue is that you are overriding Next.js' priority calculation, which is based on viewport "closeness", so you kinda need to account for that yourself. https://nextjs.org/docs/api-reference/next/image#priority

Perhaps you want to control this prop instead: https://nextjs.org/docs/api-reference/next/image#lazyboundary

**Reply by warp9pnt9** [reply]
LazyBoundary removed in Next.js v13.  Without priority, image above the fold flickers on load.  With priority, at least one wrong sized image  is preloaded and never used.  What other options do we have to resolve this?

**Comment by 1alexvash** [comment]
@icyJoseph 
If I don't apply `priority` at all, then I have another warning:
![image](https://user-images.githubusercontent.com/46556479/197133664-e38ab9a3-b6d9-4668-8f0b-e68a39c6f497.png)
Which recommends me to set it

I tried playing with lazy boundary, I'm not sure it works
